litellm/_service_logger.py

```diff
@@ -139,8 +145,7 @@ async def async_service_success_hook(
                     end_time=end_time,
                     event_metadata=event_metadata,
                 )
-            elif callback == "otel":
-                from litellm.integrations.opentelemetry import OpenTelemetry
+            elif callback == "otel" or isinstance(callback, OpenTelemetry):
                 from litellm.proxy.proxy_server import open_telemetry_logger
```

Comment:
Hi @krrishdholakia ,
this change is triggering an error if the OpenTelemetry dependency is not installed. Is not possible to check the isinstance only if callback is "otel"?